### PR TITLE
added default frequency to 1 and start date to current date to food m…

### PR DESCRIPTION
**Pull request by icpmacdo**
…odel

### CODE REVIEW

File: server/models/food.js [hunk 1]
```diff
@@ -14,11 +14,11 @@ const FoodItemSchema = new Schema({
     default: 0
   },
   startDate: {
-    type: Date
+    type: Date.now
```
Comment (jspaine) [inline server/models/food.js]: `type` should still be `Date`, just the default should be `Date.now`

File: server/models/food.js [hunk 2]
```diff
@@ -1,43 +1,44 @@
...
-  },
-  frequency: {
-    type: Number,
-    default: 0
-  },
-  deleted: {
-    type: Boolean,
-    default: false
-  }
+    name: {
```
Comment (jspaine) [inline server/models/food.js]: Looks good, but now the indentation's wrong. You might have to set your editor to use 2 spaces for tabs.

**Comment by icpmacdo** [comment]
Hey, I think the indentation is not correct. Need to figure out why VS Code wants to screw it up.

**Comment by jspaine** [comment]
Cool, thanks. It's just the default I think, you can add `"editor.tabSize": 2,` to your user settings (or workspace settings if you only want it like that for this).

**Pull request merged**